code/game/gamemodes/game_mode.dm

```diff
@@ -393,6 +391,8 @@
 			continue
 		if(is_total_antag_banned(player.ckey))
 			continue
+		if(role == ROLE_BANDIT && isseelie(player.client.prefs.pref_species.type))
```

Comment:
```suggestion
		if(role == ROLE_BANDIT && isseelie(player.client?.prefs?.pref_species?.type))
```
If we runtime here Antag selection will break- so we have to be defensive

Reply:
Got it